docs/CLI.md

```diff
@@ -36,7 +36,7 @@ After doing so, the next time you run an evergreen command that requires authent
 To submit a patch, run this from your local copy of the mongodb/mongo repo:
 
 ```bash
-evergreen patch -p <project-id>
+evergreen patch
```

Comment:
nit: passing -p is still the more straightforward practice so I have a pref to keep the doc the way it was.